scripts/db_backup.sh

```diff
@@ -322,13 +390,18 @@ cleanup_old_remote_files()
 # Parse script argument(s)
 parse_args $@
 
+# debug mode support
+if [[ $debug_mode == 1 ]]; then
+    set -x
+fi
+
 # log() and other functions
 source_wrapper $UTILITIES_FILE
 
 # Script self-idenfitication
 print_script_header
 
-log "Begin execution of $DATABASE_TYPE backup script using ${DATABASE_TYPE}dump command."
+log "Begin execution of $DATABASE_TYPE backup script using '${DATABASE_TYPE}' dump command."
```

Comment:
wrapping this in single quotes will prevent the variable replacement. in other words, ${DATABASE_TYPE} is going to be logged instead of the value assigned to DATABASE_TYPE (like mongo or mysql)

Reply:
In testing, it put single quotes around the database type:  ` Begin execution of mysql backup script using 'mysql' dump command.`